src/Components/Article/index.jsx

```diff
@@ -4,10 +4,13 @@ import SideBar from "./SideBar";
 import { Button } from "antd";
 import { PlusCircleOutlined } from "@ant-design/icons";
 import ArticleSection from "./ArticleSection";
-import NotFound from "../common/NotFound";
 
 const index = () => {
-  return <NotFound />;
+  return (
+    <>
+      <ArticleSection />
+    </>
+  );
```

Comment:
@LokeshKumar-Dev These fragments are useles, you can directly use `<ArticleSection/>`
```suggestion
  return (
      <ArticleSection />
  );
```